lib/trento_web/openapi/api_spec.ex

```diff
@@ -139,34 +144,46 @@ defmodule TrentoWeb.OpenApi.ApiSpec do
           }
         end
       end
+    end
+  end
 
-      defp build_paths_for_version(version, router) do
-        available_versions = router.available_api_versions()
+  def build_paths_for_version("latest", router) do
```

Comment:
suggestion: as the number of functions continues to increase, I think it'd be useful to have some comments explaining what each piece of code is doing.